data/coalition/coalition missions.txt

```diff
@@ -1392,3 +1392,271 @@ mission "Arachi Cartoonist 2"
+	source "Saros"
+	destination "Flowing Fields"
+	on offer
+		conversation
+			`Once you land on <origin>, Vellorae hands you 34,500 credits, and checks with a local tour line for the easiest path to the shrine. "Kindly wait for me, would you, Captain? Take a few hours, I might." She says. "Of course, if to come with me, you wish, happy to have your company, I would be."`
+			choice
+				`	"If you're okay with me going, I'd be glad to accompany you."`
+				`	"Sorry, I'm not in the mood for a mourning ritual."`
+					goto skip
+			`	Together you follow the streets out of the spaceport city and into a park filled with several different forms of trees twisting upon themselves, some reaching far up to the skies while others anchor most of their branches into the ground. The trail brings you to a large clearing, where dozens of Saryds are gathered by a circle of tall, rocky pillars with an altar at the center. Many of the Saryds there are dressed in delicate fabrics of dark greens or ochre colors and grouped into pairs that seem to be young couples, while others appear to simply be passing visitors to the shrine.`
+			`	Once the sun peeks over the tree tops and into the clearing, the visitors move away from the pillars and the alter while the couples line up and walk to the altar one at a time. There they remain for a few minutes, speaking to one another, before moving aside for the next pair. "Exchanging their vows, they are," Vellorae explains. "Our most popular wedding locations, the shrines here on the capital are."`
+			choice
+				`	"I thought this was a shrine for mourning the dead?"`
+					goto shrine
+				`	"Popular? In the middle of the woods, by some rocks?"`
+			`	She looks at you in confusion, then chuckles. "The creator of all forests, Aul Kell is, so naturally inside forests, his shrines are. Odd, I suppose they may look, to one outside our culture."`
+				goto ritual
+			label shrine
+			`	"Attributed with the gift of sunlight, Aul Kell is. As such, seek his blessings in the shrines, for many occasions we do. Only some of them, mourning and espousals are."`
+			label ritual
+			`	The couples keep exchanging their vows, until just before midday, when the sun is nearly over the central altar. At this point, the Saryds all look around the clearing, making space for Vellorae as they see her approaching the altar. She places the bangle atop the altar so it catches the sun's light, reflecting it throughout the clearing. Vellorae then begins walking around the altar, repeatedly circling it as the sun climbs higher, until it finally reaches its peak and the sun's reflection on the gem is strongest. At this point she approaches the altar again and covers the bangle with her hands. She remains there silently for a moment, then she retrieves the bangle and leaves the shrine. As she walks past, the other Saryds give her sympathetic looks, and one of the younger ones even approaches to offer her condolences before Vellorae approaches you again.`
```

Comment:
I wanted to reduce the usage of 'altar' so that it doesn't get overused.
```suggestion
			`	The couples keep exchanging their vows, until just before midday, when the sun is nearly over the central altar. At this point, the Saryds all look around the clearing, making space for Vellorae as they see her approaching the shrine. She places the bangle atop the altar so it catches the sun's light, reflecting it throughout the clearing. Vellorae then begins walking around the altar, repeatedly circling it as the sun climbs higher, until it finally reaches its peak and the sun's reflection on the gem is strongest. At this point she approaches the altar again and covers the bangle with her hands. She remains there silently for a moment, then she retrieves the bangle and leaves the shrine. As she walks past, the other Saryds give her sympathetic looks, and one of the younger ones even approaches to offer her condolences before Vellorae approaches you again.`
```